src/Map/assets/dist/abstract_map_controller.d.ts

```diff
@@ -35,7 +35,7 @@ export default abstract class<MapOptions, Map, MarkerOptions, Marker, InfoWindow
     protected map: Map;
     protected markers: Array<Marker>;
     protected infoWindows: Array<InfoWindow>;
-    initialize(): void;
+    protected additionalEventPayload: Record<string, unknown>;
```

Comment:
Hmm.. Not sure about this one. Why don't the concrete class add there aditional data in dispatch or concrete methods ? At least probably could improve the name (what event ?)

Reply:
I thought about that solution too, I can make this method abstract protected:
```typescript
    private dispatchEvent(name: string, payload: Record<string, unknown> = {}): void {
        this.dispatch(name, { prefix: 'ux:map', detail: payload });
    }
```